src/test/java/tools/jackson/dataformat/xml/failing/XmlWrapperRecord517Test.java

```diff
@@ -1,4 +1,4 @@
-package tools.jackson.dataformat.xml.records.failing;
+package tools.jackson.dataformat.xml.failing;
```

Comment:
Let's keep "records" package to avoid adding stuff at main level. I can rename I think.

Reply:
Actually it's failing test, n/m.